core/auth/fake/module.go

```diff
@@ -0,0 +1,80 @@
+package fake
+
+import (
+	"flamingo.me/dingo"
+	"flamingo.me/flamingo/v3/core/auth"
+	"flamingo.me/flamingo/v3/core/auth/fake/interfaces"
+	"flamingo.me/flamingo/v3/framework/config"
+	"flamingo.me/flamingo/v3/framework/web"
+)
+
+// Module provides Fake OpenID Connect support
+type (
+	Module struct{}
+
+	routes struct {
+		fakeController *interfaces.IdpController
+	}
+)
+
+// Interface compliance compile type checks
+var (
+	_ dingo.Module           = (*Module)(nil)
+	_ dingo.Depender         = (*Module)(nil)
+	_ config.CueConfigModule = (*Module)(nil)
+
+	_ web.RoutesModule = (*routes)(nil)
+)
+
+// Configure dependency injection
+func (*Module) Configure(injector *dingo.Injector) {
+	web.BindRoutes(injector, new(routes))
+
+	injector.BindMap(new(auth.RequestIdentifierFactory), "fake").ToInstance(interfaces.FakeIdentityProviderFactory)
+}
+
+// Inject injects routed dependencies
+func (r *routes) Inject(fakeController *interfaces.IdpController) *routes {
+	r.fakeController = fakeController
+
+	return r
+}
+
+// CueConfig schema
+func (*Module) CueConfig() string {
+	return `
+core: auth: {
+	UserConfig :: {
+		password: string | *""
+		otp: string | *""
+	}
+
+	fake :: {
+		typ: "fake"
+		broker: string
+		loginTemplate: string | *"" 
```

Comment:
`""` is not a good default if you want to indicate optional fields. Use `loginTemplate?: string` to mark as optional